README.md

```diff
@@ -110,7 +110,109 @@ The last published version of the package become [available for download](htt
 
 ### Examples
 
-> **Will come soon**
+Below are spome examples of JavaScript IPFS in action.
+
+#### Startup (in Node.js or browser)
```

Comment:
Let's break the "Creating an instance" into two parts

"Create an IPFS instance - Node.js"
"Create an IPFS instance - Browser"

Just to avoid that 'scary' require in line 121